examples/metrics/histogram.zig

```diff
@@ -0,0 +1,181 @@
+const std = @import("std");
+const sdk = @import("opentelemetry-sdk");
+const MeterProvider = sdk.MeterProvider;
+const view = sdk.View;
+const Kind = sdk.Kind;
+
+/// Histogram Example for OpenTelemetry Zig SDK
+///
+/// This example demonstrates how to use histogram instruments with different aggregation types:
+/// 1. Explicit Bucket Histograms - Uses predefined bucket boundaries
+/// 2. Exponential Bucket Histograms - Uses exponentially-sized buckets for wide value ranges
+///
+/// Histograms are ideal for measuring distributions of values like request durations,
+/// response sizes, or any metric where you want to understand the distribution pattern.
+pub fn main() !void {
+    // Allocate memory for the metrics SDK
+    const buf = try std.heap.page_allocator.alloc(u8, 4 << 20);
```

Comment:
nit: avoid magic numbers
```suggestion
    const METRICS_BUFFER_SIZE = 4 * 1024 * 1024; // 4MB buffer for metrics collection
    const buf = try std.heap.page_allocator.alloc(u8, METRICS_BUFFER_SIZE);
```